samples/bookinfo/platform/kube/rbac/upgradeToClusterRbacConfig.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+set -u
+
+# This script converts RbacConfig to ClusterRbacConfig and deletes the RbacConfig after ClusterRbacConfig
+# is successfully applied. The RbacConfig is deprecated by ClusterRbacConfig.
+
+RBAC_CONFIGS=$(kubectl get RbacConfig --all-namespaces --no-headers --ignore-not-found)
+if [ "${RBAC_CONFIGS}" == "" ]
+then
+  echo "RbacConfig not found"
+  exit 0
+fi
+
+RBAC_CONFIG_COUNT=$(echo "${RBAC_CONFIGS}" | wc -l)
+if [ "${RBAC_CONFIG_COUNT}" -ne 1 ]
+then
+  echo "${RBAC_CONFIGS}"
+  echo "found ${RBAC_CONFIG_COUNT} RbacConfigs, expecting only 1"
+  exit 1
```

Comment:
What does user need to do in this case?

Reply:
User should delete extra RbacConfigs and run this script again. Updated the prompt.